src/main/java/frc/trigon/robot/utilities/mechanisms/DoubleJointedArmMechanism2d.java

```diff
@@ -0,0 +1,219 @@
+package frc.trigon.robot.utilities.mechanisms;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.wpilibj.smartdashboard.Mechanism2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismLigament2d;
+import edu.wpi.first.wpilibj.smartdashboard.MechanismRoot2d;
+import edu.wpi.first.wpilibj.util.Color8Bit;
+import org.littletonrobotics.junction.Logger;
+
+/**
+ * A Mechanism2d object to display the current angle and the target angle of a double jointed arm.
+ */
+public class DoubleJointedArmMechanism2d {
+    private final String key;
+    private final Mechanism2d mechanism;
+    private final MechanismLigament2d
+            currentPositionFirstLigament,
+            currentPositionSecondLigament,
+            targetPositionFirstLigament,
+            targetPositionSecondLigament;
+
+    public DoubleJointedArmMechanism2d(String key) {
+        this(key, MechanismConstants.BLUE);
+    }
```

Comment:
No need for the default constructor. We'll want to always input a color since each mechanism will have a different one.